[core]/es_extended/server/functions.lua

```diff
@@ -259,6 +259,36 @@ function ESX.GetExtendedPlayers(key, val)
 	return xPlayers
 end
 
+function ESX.GetNumPlayers(key, val)
+    if(not key) then
+        return #ESX.GetPlayers()
+    end
+
+    if type(val) == "table" then
+        local numPlayers = {}
+        if key == "job" then
+            for _, v in ipairs(val) do
+                numPlayers[v] = (ESX.JobsPlayerCount[v] or 0)
+            end
+            return numPlayers
+        end
+
+        local filteredPlayers = ESX.GetExtendedPlayers(key, val)
+        for i, v in pairs(filteredPlayers) do
+            numPlayers[i] = (#v or 0)
+        end
+        return numPlayers
+    end
+
+    if key == "job" then
+        return (ESX.JobsPlayerCount[val] or 0)
+    end
+
+    local filteredPlayers = ESX.GetExtendedPlayers(key, val)
```

Comment:
#ESX.GetExtendedPlayers(key, val) because we don't use the filteredPlayers variable anywhere afterwards.

Reply:
ok